test/Bridges/Variable/zeros.jl

```diff
@@ -0,0 +1,121 @@
+
+err = ErrorException(
+    "Cannot `VectorOfVariables`-in-`MathOptInterface.Zeros` for" *
+    " which some variables are bridged but not the first one" *
+    " `MathOptInterface.VariableIndex(12345679)`."
+)
+@test_throws err MOI.add_constraint(bridged_mock, MOI.VectorOfVariables([x, y]), MOI.Zeros(2))
+
+err = ErrorException(
+    "Cannot unbridge function because some variables are bridged by" *
+    " variable bridges that do not support reverse mapping, e.g.," *
+    " `ZerosBridge`."
+)
+@test_throws err MOI.get(bridged_mock, MOI.ObjectiveFunction{typeof(obj)}())
+# With `c1`, the function does not contain any variable so it tests that it
+# also throws an error even if it never calls `variable_unbridged_function`.
+@test_throws err MOI.get(bridged_mock, MOI.ConstraintFunction(), c1)
+@test_throws err MOI.get(bridged_mock, MOI.ConstraintFunction(), c2)
+
+err = ArgumentError(
+    "Variable bridge of type `MathOptInterface.Bridges.Variable.ZerosBridge{Float64}`" *
+    " does not support accessing the attribute `MathOptInterface.Test.UnknownVariableAttribute()`."
+)
+@test_throws err MOI.get(bridged_mock, MOIT.UnknownVariableAttribute(), y)
+
+@testset "Results" begin
+    MOIU.set_mock_optimize!(mock,
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(
+            mock, [1.0],
+            (MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}) => 0.0,
+            (MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}) => 1.0)
+    )
+    MOI.optimize!(bridged_mock)
+    @test MOI.get(bridged_mock, MOI.VariablePrimal(), x) == 1.0
+    @test MOI.get(bridged_mock, MOI.VariablePrimal(), y) == 0.0
+    @test MOI.get(bridged_mock, MOI.VariablePrimal(), z) == 0.0
+
+    @test MOI.get(bridged_mock, MOI.ConstraintPrimal(), cyz) == zeros(2)
+
+    @test MOI.get(bridged_mock, MOI.ConstraintDual(), cx) == 0.0
+    @test MOI.get(bridged_mock, MOI.ConstraintDual(), c1) == 0.0
+    @test MOI.get(bridged_mock, MOI.ConstraintDual(), c2) == 1.0
+
+    err = ArgumentError(
+        "Bridge of type `MathOptInterface.Bridges.Variable.ZerosBridge{Float64}`" *
+        " does not support accessing the attribute" *
+        " `MathOptInterface.ConstraintDual(1)`."
+    )
+    @test_throws err MOI.get(bridged_mock, MOI.ConstraintDual(), cyz)
+end
+
+@testset "Query" begin
+    @test MOI.get(bridged_mock, MOI.ConstraintFunction(), cyz).variables == yz
+    @test MOI.get(mock, MOI.NumberOfVariables()) == 1
+    @test MOI.get(mock, MOI.ListOfVariableIndices()) == [x]
+    @test MOI.get(bridged_mock, MOI.NumberOfVariables()) == 3
+    @test MOI.get(bridged_mock, MOI.ListOfVariableIndices()) == [x, y, z]
+    @test MOI.get(mock, MOI.NumberOfConstraints{MOI.VectorOfVariables, MOI.Zeros}()) == 0
+    @test MOI.get(bridged_mock, MOI.NumberOfConstraints{MOI.VectorOfVariables, MOI.Zeros}()) == 1
+    @test MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MOI.VectorOfVariables, MOI.Zeros}()) == [cyz]
+end
```

Comment:
It could help to have two `test_models_equal` tests (like [here](https://github.com/JuliaOpt/MathOptInterface.jl/blob/ecf691545e67552ff437ed26ec4ddfff03c50327/test/Utilities/parser.jl#L50)), one for `bridged_mock` and one for `mock`. This would make it easy to see exactly what the bridge is doing.